app/src/test/java/org/mozilla/fenix/settings/quicksettings/protections/cookiebanners/CookieBannerHandlingDetailsViewTest.kt

```diff
@@ -134,7 +134,7 @@ class CookieBannerHandlingDetailsViewTest {
         val appName = testContext.getString(R.string.app_name)
         val expectedText =
             testContext.getString(
-                R.string.reduce_cookie_banner_details_panel_description_on_for_site,
+                R.string.reduce_cookie_banner_details_panel_description_on_for_site_1,
                 appName,
                 appName,
```

Comment:
Here you don't need any more the second appName because the string was changed.

Reply:
Thanks for catching this ❤️ !

Reply:
Addressed!